### ISSUE TITLE
bug => client keep getting disconnect

### ISSUE THREAD

**Issue by ahmedRSA** [asternic/wuzapi#12]
i have 20-30 account connect and some of them keep getting disconect

**Comment by punyahenry** [comment]
probably issue when write to sqlite file

**Comment by ahmedRSA** [comment]
> probably issue when write to sqlite file

yes maybe because we using same connection to write/read for all users 
to fix this we have to use different connection for each user
because of this issue we also get `database is locked (5) (SQLITE_BUSY)` error

**Comment by ahmedRSA** [comment]
and i think @asternic quit working on this project so its better to build your own API or find some other project built on whatsmeow

**Comment by ahmedRSA** [comment]
last commit was in 2 dec 2022 (74 days)

**Comment by asternic** [comment]
I created a new branch "one_store_per_client" that will create a different store (and connection) per whatsapp client. You are free to test it out and see if it solves the client disconnecting problem.

In any case, debug log is important to actually know if the probelm is locking on the sqlite file or something different. Without that, is just a guess game.
